From 157ff75bb5a4576a4e6a5c1fb94380c8c8b98688 Mon Sep 17 00:00:00 2001
From: Arne Coucheron <arco68@gmail.com>
Date: Sun, 26 Aug 2018 04:04:08 +0200
Subject: [PATCH 13/13] libbt-vendor: Add missing vendor prefix to a
 bluetooth.status prop

CAF forgot to add the prefix here.

Change-Id: Iac6b74edb411c5679256884b5adfe72f1a16c507
---
 libbt-vendor/src/hardware.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libbt-vendor/src/hardware.c b/libbt-vendor/src/hardware.c
index 3df76c4..50c260b 100644
--- a/libbt-vendor/src/hardware.c
+++ b/libbt-vendor/src/hardware.c
@@ -83,7 +83,7 @@ int hw_config(int nState)
 int readTrpState()
 {
     char szBtStatus[PROPERTY_VALUE_MAX] = {0, };
-    if(property_get("bluetooth.status", szBtStatus, "") < 0){
+    if(property_get("vendor.bluetooth.status", szBtStatus, "") < 0){
         ALOGE("Fail to get bluetooth status");
         return FALSE;
     }
-- 
2.17.1

